README.md

```diff
@@ -661,6 +661,7 @@ Table of Contents
   * [whimsical.com](https://whimsical.com/) - Collaborative flowcharts, wireframes, sticky notes and mind maps. Create up to 4 free boards.
   * [lensdump.com](https://lensdump.com/) - Free cloud image hosting
   * [Adobe XD](https://www.adobe.com/products/xd.html) - Wireframe & Prototyping tool similar to Sketch. Free plan covers: 1 active shared design spec, Adobe Fonts Free (limited set of fonts), 2GB of cloud storage
+  * [Krita](https://krita.org) - Free, open source painting program with a photoshop-like interface, supporting imports and exports for most image formats, including Photoshop PSD files.
```

Comment:
looks great, but unfortunately this list is not for software you run on your own computer - except developer IDEs.